### ISSUE TITLE
software-evolution-library/test fails with "return for unknown block: REST-SERVER"

### ISSUE THREAD

**Issue by quicklisp** [GrammaTech/sel#8]
Log is here: http://report.quicklisp.org/2019-03-08/failure-report/sel.html#software-evolution-library_test

**Comment by eschulte** [comment]
I believe this is because this depends on the `define-command` macro which wasn't available until recently in the https://github.com/fare/command-line-arguments package.  Is it possible to try again with the latest version of that package available?

**Comment by quicklisp** [comment]
I'm using the latest git version of command-line-arguments to test sel.

**Comment by eschulte** [comment]
Thanks for the followup.

I was able to reproduce this error in a completely empty Arch Linux image (Docker image).  Installing the minimum required to build SEL (`pacman -Syu sbcl wget git base-devel libffi`).  Installing quicklisp.  Then running `ql:register-local-projects` and trying to `ql:quickload` SEL.

I then cloned the latest command-line-arguments into `local-projects` with `git clone https://github.com/fare/command-line-arguments.git` and was able to successfully
```
* (ql:register-local-projects)
NIL
* (ql:quickload :software-evolution-library/test)
```
Given this fixed the problem and that the error suggests that the `define-command` macro is not defined in the image throwing that error I'm not sure what else I should change on my end.  Am I missing something, or is it not possible to build code which has a cross-project dependency on a version which itself is not yet in quicklisp?
